security/docker/Dockerfile.node-agent-k8s

```diff
@@ -1,11 +1,11 @@
 # BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
 ARG BASE_DISTRIBUTION=default
 
+# Version is the base image version from the TLD Makefile
+ARG BASE_VERSION=latest
+
 # The following section is used as base image if BASE_DISTRIBUTION=default
-# hadolint ignore=DL3006
-FROM istionightly/base_debug as default
-# hadolint ignore=DL3005,DL3008
-RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates && update-ca-certificates && apt-get clean  && rm -rf /var/lib/apt/lists/*
```

Comment:
we don't need these?

Reply:
No: `ca-certificates` is in the base image, and there is no such package as `update-ca-certificates`.  When I added `update-ca-certificates` to the base image, apt failed to find it.

Reply:
I'm going back to bed. :)

Reply:
this is resolved.